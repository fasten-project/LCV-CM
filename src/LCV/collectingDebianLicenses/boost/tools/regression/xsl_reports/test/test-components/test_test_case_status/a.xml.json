{
    "checksum": "0cefaa96b4a9f6177fe23b49b745767f12364f7f0cc6b73309259501458d1905",
    "file": "a.xml",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/xml"
    },
    "number_of_duplicates": 1,
    "package": "boost",
    "path": "boost/1.34.1-14/tools/regression/xsl_reports/test/test-components/test_test_case_status/a.xml",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 259566,
        "license": "/copyright/license/boost/1.34.1-14/",
        "metric": {
            "size": 116412
        },
        "pts_link": "https://tracker.debian.org/pkg/boost",
        "sloc": [
            [
                "cpp",
                642395
            ],
            [
                "xml",
                56450
            ],
            [
                "python",
                17612
            ],
            [
                "ansic",
                14520
            ],
            [
                "sh",
                2265
            ],
            [
                "yacc",
                858
            ],
            [
                "perl",
                481
            ],
            [
                "makefile",
                478
            ],
            [
                "lex",
                94
            ],
            [
                "sql",
                74
            ],
            [
                "csh",
                6
            ]
        ],
        "suites": [
            "lenny"
        ]
    },
    "raw_url": "/data/main/b/boost/1.34.1-14/tools/regression/xsl_reports/test/test-components/test_test_case_status/a.xml",
    "stat": {
        "perms": "rw-r--r--",
        "size": 550,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "1.34.1-14"
}