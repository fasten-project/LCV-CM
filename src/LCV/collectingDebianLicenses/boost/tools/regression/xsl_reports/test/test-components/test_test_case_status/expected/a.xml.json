{
    "checksum": "bd93d004644fc854f84fe34e72849f1bbb1257b4876fe508a9606f1226ebcee3",
    "file": "a.xml",
    "mime": {
        "encoding": "us-ascii",
        "type": "text/xml"
    },
    "number_of_duplicates": 1,
    "package": "boost",
    "path": "boost/1.34.1-14/tools/regression/xsl_reports/test/test-components/test_test_case_status/expected/a.xml",
    "pkg_infos": {
        "area": "main",
        "copyright": true,
        "ctags_count": 259566,
        "license": "/copyright/license/boost/1.34.1-14/",
        "metric": {
            "size": 116412
        },
        "pts_link": "https://tracker.debian.org/pkg/boost",
        "sloc": [
            [
                "cpp",
                642395
            ],
            [
                "xml",
                56450
            ],
            [
                "python",
                17612
            ],
            [
                "ansic",
                14520
            ],
            [
                "sh",
                2265
            ],
            [
                "yacc",
                858
            ],
            [
                "perl",
                481
            ],
            [
                "makefile",
                478
            ],
            [
                "lex",
                94
            ],
            [
                "sql",
                74
            ],
            [
                "csh",
                6
            ]
        ],
        "suites": [
            "lenny"
        ]
    },
    "raw_url": "/data/main/b/boost/1.34.1-14/tools/regression/xsl_reports/test/test-components/test_test_case_status/expected/a.xml",
    "stat": {
        "perms": "rw-r--r--",
        "size": 246,
        "symlink_dest": null,
        "type": "-"
    },
    "text_file": true,
    "type": "file",
    "version": "1.34.1-14"
}